src/custom-rules/max-lines-per-function-jsx.js

```diff
@@ -0,0 +1,230 @@
+// Copied from https://github.com/eslint/eslint/blob/e52b98bf25d882da4efd5559ce5974b6697cf701/lib/rules/max-lines-per-function.js
```

Comment:
Is it copied 100% or was it modified? If the latter, it would be good to mark the place of modification.